src/components/NewGalaxyModal.js

```diff
@@ -119,23 +119,23 @@ export const NewGalaxyModal = _.flow(
 
   const renderDefaultCase = () => {
     const { cpu, memory } = _.find({ name: 'n1-standard-8' }, machineTypes)
-    const cost = hourlyKubernetesAppCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-standard-8' } }) + persistentDiskCost({ size: 30, status: 'Running' })
+    const cost = getGalaxyCost(app)
```

Comment:
I think the `app || ...` expression should remain here, since it's specific to this UI (we're either reporting on an existing app, or showing a preview of what the cost will be after creation). That way the helper function can always expect a value.